procfs/proc.go

```diff
@@ -34,40 +34,75 @@ type Procer interface {
 	NewProcProc() ([]ProcProc, error)
 }
 
-//NewProcProc collects data from various proc pseudo-file system files
-//and converts it into a ProcProc structure.
+var (
```

Comment:
You'll want to slap mutexes on this block just in case. If possible, I would store these as state in a struct somewhere instead of at the package level.